misc/random.c

```diff
@@ -44,6 +47,16 @@ void mp_rand_seed(uint64_t seed)
 #ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
     seed = 42;
 #endif
+
+    if (seed == 0) {
+        char buf[sizeof(seed)];
+        if (av_random_bytes((uint8_t *)&buf, sizeof(buf)) < 0) {
```

Comment:
I missed it before but we can get rid of the cast too
```suggestion
        uint8_t buf[sizeof(seed)];
        if (av_random_bytes(buf, sizeof(buf)) < 0) {
```